server/session/player.go

```diff
@@ -227,7 +227,7 @@ func (s *Session) Transfer(ip net.IP, port int) {
 // SendGameMode sends the game mode of the Controllable entity of the session to the client. It makes sure the right
 // flags are set to create the full game mode.
 func (s *Session) SendGameMode(mode world.GameMode) {
-	flags, id, perms := uint32(0), int32(packet.GameTypeSurvivalSpectator), uint32(0)
+	flags, id, perms := uint32(0), int32(packet.GameTypeSurvival), uint32(0)
```

Comment:
Have you tested that this does not result in any broken gamemodes? What's the reason for this change?

Reply:
This was a mistake I made in the v1.17.0 update, this was always meant to be GameTypeSurvival. It seemed to have not caused any issues until implementing bows, as it causes the bow to always release client side.

Reply:
If you back a few DF versions, you'll see that this was also GameTypeSurvival, as it should have been.